src/contentScript/sidebarDomControllerLite.ts

```diff
@@ -27,9 +27,12 @@ import { uuidv4 } from "@/types/helpers";
 
 export const SIDEBAR_WIDTH_CSS_PROPERTY = "--pb-sidebar-width";
 const ORIGINAL_MARGIN_CSS_PROPERTY = "--pb-original-margin-right";
+const EASING_FUNCTION = "cubic-bezier(0.23, 1, 0.32, 1)";
+const TRANSITION_DURATION = "500ms";
```

Comment:
```suggestion
const TRANSITION_DURATION = 500;
```

<img width="667" alt="Screenshot 3" src="https://github.com/pixiebrix/pixiebrix-extension/assets/1402241/c19892a3-98b5-4b54-a0f3-39a868ba5caf">

Reply:
Interesting, I missed the error in the console, but the animation still works. Originally had it as just 500, but I changed it for the other locations I used it. I'll revert.